8.cytopick_analysis/scripts/prediction_images_find.py

```diff
@@ -0,0 +1,532 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook finds random cells from each prediction category and displays them. The purpose is to get representative images examples of each category.
+
+# In[1]:
+
+
+import pathlib
+
+import cv2
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+# import pillow and open cv
+import PIL
+import seaborn as sns
+import tifffile as tf
+from cytocherrypick import cherrypick
+from PIL import Image, ImageEnhance
+from tqdm import tqdm
```

Comment:
Recommend including code comment for this import to describe what it is doing (I am just a bit confused, but please disregard if you think it is not needed).

Reply:
Updated!